app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User <ApplicationRecord
   validates_presence_of :address, presence: true
   validates_presence_of :city, presence: true
   validates_presence_of :state, presence: true
-  validates_presence_of :zip, presence: true
+  validates_presence_of :zip, presence: true, numericality: { only_integer: true, message: "must be a number" }
```

Comment:
Could change syntax here to match the rest of our validations. Maybe something like `validates_numericality_of :zip, only_integer: true`.  I'm also not entirely sure why presence: true is added on the end of the presence validations... seems kind of redundant but I'm not entirely clear on it. Maybe table it for now if it's working